src/components/Modal.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Modal extends React.Component {
+  static propTypes = {
+    onClose: PropTypes.func,
+    show: PropTypes.bool,
+    className: PropTypes.string,
+    title: PropTypes.string,
+    children: PropTypes.node,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (typeof document !== 'undefined')
+      document.body.classList[this.props.show ? 'add' : 'remove']('showModal');
```

Comment:
I'm surprised we don't have a lint rule for that (or prettier), anyway, put braces here.

Background: https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/